docs/security.rst

```diff
@@ -0,0 +1,34 @@
+Security Policy
+===============
+
+This documents the security policy and actions to take to secure the security of the package, its deployment and use.
+
+Supported Versions
+------------------
+
+Security vulnerabilities are fixed only for the latest version of `icalendar`.
+
+.. list-table:: Versions to receive security updates
+   :widths: 25 25
+   :header-rows: 1
+
+   * - Version
+     - Supported
+   * - 6.*
+     - ✅
+   * - 5.*
+     - ❌
+   * - 4.*
+     - ❌
+   * - < 4.*
+     - ❌
+
+
+Reporting a Vulnerability
+-------------------------
+
+Please `report vulnerabilities of icalendar to Plone
+<https://github.com/plone/.github/blob/main/SECURITY.md#readme>`_.
```

Comment:
Do not refer to links that refer to links, especially if you cannot honor the policies to which they refer. You should instead grab the relevant bits that you want to follow and incorporate them into your custom security policy.

As this states, eventually people may report the security issue to security@plone.org. Are you a member of that email group? If not, you would need to discuss with them how to handle reports for icalendar sent to that address. It may be better to use another address.

Reply:
We can activate GitHub's security reporting for this project.

I wonder: If a vulnerability gets reported on GitHub, then the other Plone packages do not know it exists.
If a vulnerability is reported to Plone, then I do not know it was reported.

Thus, I wonder what to do. We have two different groups here that are affected. I asked the security team to enlighten us here.

Also, nobody reported anything like this for years and it is unlikely to happen. So. I wonder if this is hypothetical. But a report should also not go into the void after a few years.

Reply:
> We can activate GitHub's security reporting for this project.

To enable GitHub security, you create a `SECURITY.md` file at the root of your repo. GitHub should automatically detect it. It's nothing magical. It's mere documentation of how to report security issues, and how you will respond to them.

> I wonder: If a vulnerability gets reported on GitHub, then the other Plone packages do not know it exists. If a vulnerability is reported to Plone, then I do not know it was reported.
> 
> Thus, I wonder what to do. We have two different groups here that are affected. I asked the security team to enlighten us here.

I would follow Pylons Project's workflow, modified to your preference.

https://github.com/Pylons/.github/blob/main/SECURITY.md

To summarize:

1. Users report security issues responsibly and in a manner that allows maintainers to respond in a timely and effective manner.
2. Maintainers verify and fix.
3. All the bullet points at that link.

I have no insight into the Plone Security Team's internal process. I'm not a member, and it's not transparent. I never received a report back of what they did for an issue that I recently reported to them.

> Also, nobody reported anything like this for years and it is unlikely to happen. So. I wonder if this is hypothetical. But a report should also not go into the void after a few years.

It's only hypothetical until it's not, and then it's too late.

Reply:
Thanks for all the replies.

GitHub also allows security reporting. This is how it looks from the outside from the Open Web Calendar, my project. There is a button to report.
![grafik](https://github.com/user-attachments/assets/60f84a4b-2f31-4d03-ab42-e2185131af9a)

I like the one you showed me (Pylons). This is also nice to follow for someone (me) who did not go through this process yet but is in a position of responsibility.

- I will copy that in.
- change the links
- use githubs security report feature - mainly because I do not want to maintain an independent list of who gets notified - it should be the GitHub project's maintainers.
- I would add that we also notify the Plone security team.

Reply:
I did not know that GitHub had that feature. Your plan sounds good to me.